tool/tbot/main_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/lib/utils/golden"
+	"github.com/gravitational/teleport/tool/tbot/config"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_onConfigure(t *testing.T) {
+	t.Parallel()
+
+	// This is slightly rubbish, but due to the global nature of `botfs`,
+	// it's difficult to configure the default acl and symlink values to be
+	// the same across dev laptops and GCB.
+	// If we switch to a more dependency injected model for botfs, we can
+	// ensure that the test one returns the same value across operating systems.
+	normalizeOSDependentValues := func(data []byte) []byte {
```

Comment:
I'm really not a fan of this, but I didn't want to completely refactor `botfs` to be a dependency injected struct without a bit more investigation.

I'm completely open to some more suggestions here.

Reply:
Yeah, that's unfortunate. The overly-simple botfs design has caused some problems already and I'm not opposed to refactoring it.